base/i18n/number_formatting_unittest.cc

```diff
@@ -91,7 +91,8 @@ TEST(NumberFormattingTest, FormatDouble) {
   }
 }
 
-TEST(NumberFormattingTest, FormatPercent) {
+// TODO(b/316198056): Re-enable the test
```

Comment:
TODOs now take the format `TODO: b/123 - Do whatever` see http://go/todo-format

Reply:
Also let's say more specifically "Re-enable this test once base/net have been updated" or something along those lines